src/fluree/db/json_ld/commit_data.cljc

```diff
@@ -437,86 +437,144 @@
       add-tt-id
       (update :schema vocab/update-with* -1 commit-schema-flakes)))
 
+(defn commit-metadata-flakes
+  "Builds and returns the commit metadata flakes for the given commit, t, and
+  db-sid. Used when committing to an in-memory ledger value and when reifying
+  a ledger from storage on load."
+  [{:keys [address alias branch data id time v] :as _commit} t db-sid]
+  (let [{db-id :id db-t :t db-address :address :keys [flakes size]} data]
+    [;; link db to associated commit meta: @id
+     (flake/create t const/$xsd:anyURI id const/$xsd:string t true nil)
+
+     ;; commit flakes
+     ;; address
+     (flake/create t const/$_address address const/$xsd:string t true nil)
+     ;; alias
+     (flake/create t const/$_ledger:alias alias const/$xsd:string t true nil)
+     ;; branch
+     (flake/create t const/$_ledger:branch branch const/$xsd:string t true nil)
+     ;; v
+     (flake/create t const/$_v v const/$xsd:int t true nil)
+     ;; time
+     (flake/create t const/$_commit:time (util/str->epoch-ms time)
+                   const/$xsd:dateTime t true nil) ;; data
+     (flake/create t const/$_commit:data db-sid const/$xsd:anyURI t true nil)
+
+     ;; db flakes
+     ;; @id
+     (flake/create db-sid const/$xsd:anyURI db-id const/$xsd:string t true nil)
+     ;; t
+     (flake/create db-sid const/$_commitdata:t db-t const/$xsd:int t true nil)
+     ;; address
+     (flake/create db-sid const/$_address db-address const/$xsd:string t true nil)
+     ;; size
+     (flake/create db-sid const/$_commitdata:size size const/$xsd:int t true nil)
+     ;; flakes
+     (flake/create db-sid const/$_commitdata:flakes flakes const/$xsd:int t true
+                   nil)]))
+
+(defn prev-commit-flakes
```

Comment:
pedantic nitpick that you can take or leave, but it looks like only one flake is created here, in `prev-data-flakes`, and in `message-flakes` below

Reply:
Yeah it's plural to indicate that it returns a collection and so callers needn't necessarily care how many flakes are returned.